src/train/exporter/validator.py

```diff
@@ -7,7 +7,7 @@
 from loader import load_train_args
 from config import ERROR_KEY
 
-required_benchmark = ["stressng_kepler_query"]
+required_benchmark = ["sample_kepler_query", "stressng_kepler_query", "customBenchmark_kepler_query"]
```

Comment:
required benchmark is defined for making sure that the required benchmark is included when exporting the model. sample_kepler_query and customBenchmark_kepler_query should not be included here as it is optional.

Reply:
The setting [`MAIN_COLLECT_INPUT="stressng"`](https://github.com/sustainable-computing-io/kepler-model-server/blob/main/model_training/script.sh#L214) along with [`required_benchmark = ["stressng_kepler_query"]`](https://github.com/sustainable-computing-io/kepler-model-server/blob/main/src/train/exporter/validator.py#L10) only lets exporting models trained using the benchmark `stressng`. Ideally, it should allow any other benchmark (like `sample`, `customBenchwork` or some other future workload added to the collection of benchmarks).

Reply:
The original design is to avoid the overfitting benchmark suite. 
So, the concept of required benchmark here is to assure that stressng should be included in the benchmark suite. 
However, if we will allow any benchmark suite (no need to include stressng), we can just remove this required benchmark check.  We can add the checklist to have stressng workload manually on PR pushed to the kepler-model-db repo. 
Would you mind noting the issue to remove this requirement check on the model server source side?

Reply:
Sure @sunya-ch , did this update.